cmd/main.go

```diff
@@ -133,13 +141,17 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err = (&controller.RuntimeReconciler{
-		Client: mgr.GetClient(),
-		Scheme: mgr.GetScheme(),
-	}).SetupWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "Runtime")
-		os.Exit(1)
+	if enableProvisioning {
```

Comment:
While the RuntimeReconciler is not yet implemented, it will not only take care of provisioning but also upgrades and de-provisioning.

Could this be renamed to `enableRuntimeReconciler`?

Reply:
Received a 👍 from @akgalwas about this one, will implement it myself